README.md

```diff
@@ -60,6 +60,65 @@ private
 attr_reader :foo, :bar
 ```
 
+### Using success/failure callbacks
+
+Sometimes it is helpful for the class to return success/failure callbacks rather than just the resulting value.
+This can be achieved by including the `Mandate::Callbacks` module as follows:
+
+```ruby
+class Sumer
+  include Mandate
+  include Mandate::Callbacks
+
+  initialize_with :num1, :num2
+
+  def call
+    abort!("num1 must be an Integer") unless num1.is_a?(Integer)
+    abort!("num2 must be an Integer") unless num2.is_a?(Integer)
+
+    num1 + num2
+  end
+end
+
+cmd = Sumer.(1,2)
+cmd.success { |result| p result } # puts 3
+cmd.failure { |errors| p errors } # Noop
+cmd.succeeded? # true
+cmd.result # 3
+cmd.errors # []
+
+cmd = Sumer.("1","2")
+cmd.success { |result| p result } # Noop
+cmd.failure { |errors| p errors } # puts ["num1 must be an Integer"]
+cmd.succeeded? # false
```

Comment:
```suggestion
cmd.succeed? # false
```

Or maybe `success?`

Reply:
@kotp The code defines the method as `succeeded?`. I'm unsure if you're suggesting changing the method name, and if so why? :)

Reply:
Not sure if the name is better served as a "success?" so yes.  But the method doesn't necessarily need to be changed, but could be aliased.  If `success?` would be reflexively used.